### PR TITLE
Integrated Kubernetes configurations for Docker-Project

### PR DESCRIPTION
**Pull request by imzainhafeez**
Contains Kubernetes configuration files for mongo-db and mongo-express services for our Docker Project,

### CODE REVIEW

File: k8s/mongo-secret.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: mongodb-secret
+type: Opaque
+data:
+    mongo-root-username: dXNlcm5hbWU=
+    mongo-root-password: cGFzc3dvcmQ=
```
Comment (Abdullahxz) [inline k8s/mongo-secret.yml]: Please avoid pushing Secret manifests to version control, as they can contain sensitive information.

**Review by Abdullahxz — CHANGES_REQUESTED** [review]

File: k8s/mongo.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
```
Comment (Abdullahxz) [inline k8s/mongo.yml]: Creating a deployment for a stateful application?

**Review by Abdullahxz — CHANGES_REQUESTED** [review]
I do not see any manifests for the application. There are only database manifests. Please create them here.

File: k8s/mongo-secret.yml (hunk 1 above)
Comment (imzainhafeez) [reply k8s/mongo-secret.yml]: Sure, done.

File: k8s/mongo.yml (hunk 2 above)
Comment (imzainhafeez) [reply k8s/mongo.yml]: Added application manifests and corrected to statefulSet as needed.

**Review by Abdullahxz — CHANGES_REQUESTED** [review]
Kindly add sealed-secrets to this PR as discussed,

**Comment by imzainhafeez** [comment]
fixed.

**Review by Abdullahxz — APPROVED** [review]
Approved.

**Pull request merged**